From 3fa4094ed3abbd6b34003f28282a13b3275f2042 Mon Sep 17 00:00:00 2001
From: David Daney <ddaney@caviumnetworks.com>
Date: Thu, 3 Feb 2011 18:00:49 -0800
Subject: [PATCH 20/97] MIPS: Octeon: Change load address to waste less memory.

Source: Cavium SDK 2.1.0-407

"Modern" octeon bootloaders allow us to load the image at 0x100000.
Moving there from 0x1100000 makes 16MB more memory usable.

Signed-off-by: David Daney <ddaney@caviumnetworks.com>
Integrated-by: Phil Staub <Phil.Staub@windriver.com>
---
 arch/mips/Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/mips/Makefile b/arch/mips/Makefile
index ba98516..596eb44 100644
--- a/arch/mips/Makefile
+++ b/arch/mips/Makefile
@@ -688,7 +688,7 @@ else
 ifdef CONFIG_WRHV
 load-$(CONFIG_CPU_CAVIUM_OCTEON) 	+= 0xffffffffc0001000
 else
-load-$(CONFIG_CPU_CAVIUM_OCTEON) 	+= 0xffffffff81100000
+load-$(CONFIG_CPU_CAVIUM_OCTEON) 	+= 0xffffffff80100000
 endif
 endif
 
-- 
1.7.0

